src/test/components/__snapshots__/CallTreeSidebar.test.js.snap

```diff
@@ -44,10 +44,7 @@ exports[`CallTreeSidebar matches the snapshots when displaying data about the cu
     <div
       class="sidebar-value"
     >
-      2ms
-       (
-      50
-      %)
+      2ms (50%)
```

Comment:
The snapshot changes are because of the change to how we display the values, but nothing changes really.